cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,14 @@
+class AddressStepPage {
+  private proceedCheckout: string;
+
+  constructor() {
+    this.proceedCheckout = "button.button.btn.btn-default.button-medium";
+  }
+
+  public proceedCheckoutAuth(): void {
+    cy.get(this.proceedCheckout).click();
+  }
+}
+
+export {AddressStepPage};
+
```

Comment:
delete this line, just leave one empty line at the end of the file

Reply:
resolved